protocol/auditor/common_test.go

```diff
@@ -35,9 +32,18 @@ func TestComputeDirectoryIdentity(t *testing.T) {
 					}
 				}()
 			}
-			if got := ComputeDirectoryIdentity(tt.args.str); got != tt.want {
-				t.Errorf("ComputeDirectoryIdentity() = %v, want %v", got, tt.want)
+			if got, want := ComputeDirectoryIdentity(tt.str), tt.want; !bytes.Equal(got[:], want) {
+				t.Errorf("ComputeDirectoryIdentity() = %#v, want %#v", got, want)
 			}
 		})
 	}
 }
+
+// decode hex string to byte array
+func dh(h string) []byte {
```

Comment:
This is another one of those functions that could be exposed in a `testutil` module. I would also change the name from `dh` to something more meaningful.

Reply:
Renamed to `hex2bin`.